jobs/build/release/Jenkinsfile

```diff
@@ -138,16 +138,22 @@ node {
     direct_release_nightly = false
     detect_previous = true
     candidate_pr_only = false
+    release_offset = params.RELEASE_OFFSET.toInteger()
```

Comment:
`toInteger()` would throw exception when it's empty for hotfix.

Reply:
Nice catch! Updated.